### DISCUSSION TITLE
Communicate with Android

### DISCUSSION THREAD

**Discussion by miladnorouziii** [mikebrady/shairport-sync#1544] [Q&A]
Hi, i'm a mobile developer and i want to know is there any way to cast music to this service with an android device? 
@mikebrady I'm interested in your work, really nice, piece of cake to use with iOS .

**Comment by mikebrady** [comment]
Thanks for your interest in Shairport Sync. Since it implements AirPlay and AirPlay 2, an Android device would have to do the same thing -- emulate the AirPlay or AirPlay 2 protocols. There are some Android apps out there. Also [OwnTone](https://github.com/owntone/owntone-server#airplay-devicesspeakers) can do AirPlay, so it's worth a look.

**Comment by yadavBahubali** [comment]
Hi, I'm Software Developer and i was looking for something that helps me making connection between ios to android for Airplay (Screen mirror ,Photo Slideshow,Musics player etc). Can You help me Regarding this idea!!! @mikebrady  @miladnorouziii
